zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -106,11 +108,7 @@ public void open(final Controller controller) {
     writer = createBatchWriter();
 
     scheduleDelayedFlush();
-
-    // // start archiver after the schema has been created to avoid transient errors
-    if (configuration.getArchiver().isRolloverEnabled()) {
-      taskManager.start();
-    }
+    taskManager.start();
```

Comment:
:question: does this change the behavior of the component at startup or is the configuration always true?